packages/ember-application/lib/system/engine-instance.js

```diff
@@ -220,13 +219,12 @@ EngineInstance.reopenClass({
     registry.injection('view', '_environment', '-environment:main');
     registry.injection('route', '_environment', '-environment:main');
 
+    let rendererType = 'inert';
     if (options.isInteractive) {
-      registry.injection('view', 'renderer', 'renderer:-dom');
-      registry.injection('component', 'renderer', 'renderer:-dom');
-    } else {
-      registry.injection('view', 'renderer', 'renderer:-inert');
-      registry.injection('component', 'renderer', 'renderer:-inert');
+      rendererType = 'dom';
```

Comment:
I much prefer the code on the left, no need for additional variable and interpolations, and is quite explicit.